templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -42,20 +42,44 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>Same as above - I think there a small typo, should be <code>../data/example_dataset</code>. Also, I don't think we need quotes around <code>"../data/example_dataset"</code> , since it's already displaying as code.</p><p> </p><p>Also, I think the last sentence of the base_dir bullet point should say "...created during&nbsp;cell clustering" (not pixel clustering).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>